corehq/apps/app_manager/management/commands/set_linked_applications_uses_master_forms_true.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, print_function
+from __future__ import unicode_literals
+
+from django.core.management.base import BaseCommand
+from corehq.apps.app_manager.models import LinkedApplication, Application
+
+
+class Command(BaseCommand):
+    @staticmethod
+    def _iter_linked_apps():
+        app_ids = [
+            result['id'] for result in Application.get_db().view(
+                'app_manager/applications',
+                reduce=False,
+                include_docs=False
```

Comment:
If you `include_docs` you can just wrap the output of this command so you don't have to fetch the app again (although maybe you are doing this as a safeguard against pulling all apps into memory)